tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs

```diff
@@ -249,6 +249,21 @@ public static void RewriteRenovate(ApiCatalog catalog)
             {
                 ["extends"] = new JArray { "config:base" },
                 ["ignorePaths"] = new JArray(catalog.Apis.Select(api => $"apis/{api.Id}/{api.Id}/**").ToArray()),
+                ["packageRules"] = new JArray
```

Comment:
Hmm... I can see this getting messy. Let's keep it for now, but maybe we should make the update just load the existing JSON and update the ignorePaths property of it, leaving everything as it is. If that sounds good to you, I can do that later on.

Reply:
Yes agreed, I mostly want to check that this will stop the renovate PRs from generating. Then, or when this part gets messier, we can do differently.